### PR TITLE
Add aarch64 ubuntu bazel bot

### PR DESCRIPTION
**Pull request by gchatelet**

### CODE REVIEW

**Comment by Mizux** [comment]
@gchatelet PR should be ready for review !

https://github.com/google/cpu_features/actions/workflows/aarch64_linux_bazel.yml
https://github.com/google/cpu_features/actions/workflows/amd64_linux_bazel.yml

File: bazel/ci/Makefile [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+DOCKER_BUILDX_CMD := ${DOCKER_BUILDX_CMD} --no-cache
+endif
+ifdef VERBOSE
+DOCKER_BUILDX_CMD := ${DOCKER_BUILDX_CMD} --progress=plain
+endif
+DOCKER_RUN_CMD := docker run --rm --init --net=host
+
+# $* stem
+# $< first prerequist
+# $@ target name
```
Comment (gchatelet) [inline bazel/ci/Makefile:72]: Are these three lines necessary?

File: bazel/ci/Makefile [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+############
+## NATIVE ##
+############
+# ref: https://go.dev/doc/install/source#environment
+# ref: https://github.com/containerd/containerd/blob/269548fa27e0089a8b8278fc4fc781d7f65a939b/platforms/platforms.go#L80-L94
+PLATFORMS := amd64 arm64
+STAGES := env devel build test
+
+define make-platform-stage-target =
+#$$(info PLATFORM: '$1' STAGE: '$2')
```
Comment (gchatelet) [inline bazel/ci/Makefile:83]: remove?

File: bazel/ci/Makefile [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+$1_$2: docker/Dockerfile
+	#@docker image rm -f ${IMAGE}:$1_$2 2>/dev/null
```
Comment (gchatelet) [inline bazel/ci/Makefile:85]: remove?

File: bazel/ci/Makefile [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+sh_$1_$2: $1_$2
+	${DOCKER_RUN_CMD} --platform linux/$1 -it --name ${IMAGE}_$1_$2 ${IMAGE}:$1_$2
+
+clean_$1_$2:
+	docker image rm -f ${IMAGE}:$1_$2 2>/dev/null
+	rm -f cache/$1/docker_$2.tar
+endef
+
+define make-platform-target =
+#$$(info PLATFORM: $1)
```
Comment (gchatelet) [inline bazel/ci/Makefile:108]: remove?

File: bazel/ci/docker/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,40 @@
+# Create a virtual environment with all tools installed
+# ref: https://hub.docker.com/_/ubuntu
+FROM ubuntu:latest AS env
+
+# python3 pkgconf libglib2.0-dev \
```
Comment (gchatelet) [inline bazel/ci/docker/Dockerfile:5]: remove?

File: bazel/ci/docker/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+&& DEBIAN_FRONTEND=noninteractive apt-get install -yq \
+ git wget build-essential \
+&& apt-get clean \
+&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+ENTRYPOINT ["/usr/bin/bash", "-c"]
+CMD ["/usr/bin/bash"]
+
+# Install Bazelisk
+ARG PLATFORM
+#ENV PLATFORM ${PLATFORM:-unknown}
```
Comment (gchatelet) [inline bazel/ci/docker/Dockerfile:19]: remove?

File: scripts/generate_badges.d [hunk 7]
```diff
@@ -1,3 +1,6 @@
+// usage:
+// 1. copy paste the whole code to https://run.dlang.io/ then clic `run`
```
Comment (gchatelet) [inline scripts/generate_badges.d:2]: `click`

File: bazel/ci/Makefile (hunk 1 above)
Comment (Mizux) [reply bazel/ci/Makefile]: done

File: bazel/ci/Makefile (hunk 2 above)
Comment (Mizux) [reply bazel/ci/Makefile]: done

File: bazel/ci/Makefile (hunk 3 above)
Comment (Mizux) [reply bazel/ci/Makefile]: done

File: bazel/ci/Makefile (hunk 4 above)
Comment (Mizux) [reply bazel/ci/Makefile]: done

File: bazel/ci/docker/Dockerfile (hunk 5 above)
Comment (Mizux) [reply bazel/ci/docker/Dockerfile]: done

File: bazel/ci/docker/Dockerfile (hunk 6 above)
Comment (Mizux) [reply bazel/ci/docker/Dockerfile]: done

**Review by gchatelet — COMMENTED** [review]
LGTM ! Thx a bunch  👍

**Pull request merged**